### ISSUE TITLE
Written for Garmin device?

### ISSUE THREAD

**Issue by TomMaresca7** [chris-revell/training-diary#1]
It appears this was written to be used with a Garmin device. Is that correct? This is more of a question than an issue, but I do not see any other way to ask a question. My apologies, I am somewhat unfamiliar with github still.

**Comment by chris-revell** [comment]
That's right, although it should work with any .fit file produced by any other fitness device. I haven't quite decided what I want to do with the code long term - right now it just makes nicer graphs than the default Garmin ones (I think).

**Comment by TomMaresca7** [comment]
Thanks for that information, Mr. Revell. I have not decided what I want to do with my project, either. You have a happy New Year and stay safe there in the UK. My wife's sister lives there and she says it's been particularly difficult the past few weeks.

**Comment by chris-revell** [comment]
No problem. Yes, it’s not great here currently but we’ll survive!

**Comment by TomMaresca7** [comment]
Yes, that is true. As the saying goes, "The best way out is always through."
